src/blob_gc_job.cc

```diff
@@ -232,7 +209,7 @@ Status BlobGCJob::DoSample(const BlobFileMeta* file, bool* selected) {
 
   *selected =
       discardable_size >=
-      std::ceil(sample_size_window *
+      std::ceil(iterated_size *
```

Comment:
iterated_size should be offset than size;

Reply:
Then the blob file is not likely to gc if it has many holes.